sample-config-advanced.json

```diff
@@ -519,23 +519,23 @@
           "cert": "saml.pem"
         },
         "oidc": {
-          "authorizationURL": "https://sso.server.com/api/oidc/authorization",
-          "callbackURL": "https://mesh.server.com/oidc-callback",
-          "clientid": "00000000-0000-0000-0000-000000000000",
-          "clientsecret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-          "issuer": "https://sso.server.com",
-          "tokenURL": "https://sso.server.com/api/oidc/token",
-          "userInfoURL": "https://sso.server.com/api/oidc/userinfo",
-          "logoutURL": "https://sso.server.com/logout",
-          "newAccounts": true,
+          "issuer": {
+            "issuer": "https://sso.server.com",
+            "end_session_endpoint": "https://sso.server.com/logout"
+          },
+          "client": {
+            "clientid": "00000000-0000-0000-0000-000000000000",
+            "clientsecret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+          },
           "groups": {
             "required": [ "groupA", "groupB", "groupC" ],
             "siteadmin": [ "groupA" ],
             "sync": {
               "enable": true,
               "filter": [ "groupB", "groupC" ]
             }
-          }
+          },
```

Comment:
There should be `custom` section with `preset` and `tenant_id`.

Reply:
This is just an example of an advanced generic oidc configuration. If you want to delve into the presets and such there is documentation but I thought it would be prodent to keep that out of the example configs here. If you want to see all the options available you should be looking at the schema, not the sample configs.